analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AppSettings/DatabasePasswordsShouldBeSecure/Corrupt/appsettings.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "ConnectionStrings": {
+    "DefaultConnection": "Server=myServerAddress;Database=myDataBase;User Id=myUsername;Password="
```

Comment:
you could even remove lines 3 and 4 to make it more obvious it's corrupt

currently I had to look why it's corrup (missing brace)